.github/workflows/CI.yml

```
@@ -6,7 +6,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        node-version: [ 20 ]
+        node-version: [ 20, 22 ]
```

@barmac revert. or @nikku can i force push to your branch? (Asking because in previous projects we strongly recommended not to intefer with other branches
```suggestion
        node-version: [ 20 ]
```

---

i won't take over this PR, pushed suggestion